packages/deployments/contracts/contracts/nomad-xapps/contracts/connext/ConnextHandler.sol

```diff
@@ -294,6 +311,41 @@ contract ConnextHandler is
     );
   }
 
+  /**
+   * @notice Sets the Aave Pool contract address.
+   * @dev Allows to set the aavePool to address zero to disable Aave Portal if needed
+   * @param _aavePool The address of the Aave Pool contract
+   */
+  function setAavePool(address _aavePool) external onlyOwner {
+    aavePool = _aavePool;
+  }
+
+  /**
+   * @notice Sets the Aave Portal fee numerator
+   * @param _aavePortalFeeNumerator The new value for the Aave Portal fee numerator
+   */
+  function setAavePortalFee(uint256 _aavePortalFeeNumerator) external onlyOwner {
+    if (_aavePortalFeeNumerator > LIQUIDITY_FEE_DENOMINATOR) revert ConnextHandler__setAavePortalFee_invalidFee();
+
+    aavePortalFeeNumerator = _aavePortalFeeNumerator;
+  }
```

Comment:
ideally we could pull this from aave directly. also, i think the fees differ per-pool?

Reply:
I'm using this comment for all the comments related to fees.

Aave has only one Pool contract which is the entry point for the protocol which handles all the assets. 
In the contracts, there is no information about the portal fees, only the `BRIDGE_PROTOCOL_FEE()` which is used to [calculate](https://github.com/aave/aave-v3-core/blob/e46341caf815edc268893f4f9398035f242375d9/contracts/protocol/libraries/logic/BridgeLogic.sol#L123) the proportion of the paid fees that will be stored for the protocol instead of the Aave users. But there is no stored information about how to calculate the fees for each asset and/or for each bridge using Portals. That's why I created the `aavePortalFeeNumerator` variable.

I found this template for Portal whitelist https://governance.aave.com/t/arc-portals-governance-framework/7681
It looks like each bridge can propose a specific fee model for each asset they will use from Portals. It looks like in this V0 of Portals fees works more like a social/off-chain agreement than enforced via smart contracts. So, depending in our proposal, if we are going to use different pricing models and/or percentages on different assets, instead of storing one variable `aavePortalFeeNumerator` we may need to have a mapping relating the adopted asset a the feeNumerator.

It would be great to confirm all of this with someone from Aave.

Reply:
i think you're right -- will confirm